src/backend/joanie/core/helpers.py

```diff
@@ -58,22 +58,17 @@ def generate_certificate_for_order(order):
     return 1
 
 
-def generate_certificates_for_orders(orders):
+def generate_certificates_for_orders(orders_filtered):
```

Comment:
We don't know what order query people will pass so just call it "orders"?

Reply:
ok! instead I've renamed the variable called orders the line underneath